extensions/mssql/src/mssql.d.ts

```diff
@@ -153,114 +153,29 @@ declare module 'mssql' {
 		defaultDeploymentOptions: DeploymentOptions;
 	}
 
+	/**
+	* Interface containing deployment options of boolean type
+	*/
 	export interface DacDeployOptionPropertyBoolean {
 		value: boolean;
 		description: string;
 		displayName: string;
 	}
 
+	/**
+	* Interface containing deployment options of integer type and hold DacFx ObjectType enum values
```

Comment:
Link to exactly what enum (source would be preferable, but docs is fine too) you're referring to

Reply:
unfortunately "ObjectType" is the enum name here! :(

Reply:
What do you mean "unfortunately"? I don't have any concerns with the name. I'm saying provide a direct source link (to wheever this enum lives in the dacfx repo) in the comment so that people are able to see what the enum values to expect are.

Reply:
Sure, will add the enum location to the comment, I thought you were asking about the Enum file name!